docs/sources/write/shortcodes/index.md

```diff
@@ -26,7 +26,7 @@ The Grafana shortcode templates are defined in the `layouts/shortcodes` folder o
 To request custom shortcodes, [create an issue](https://github.com/grafana/writers-toolkit/issues).
 {{% /admonition %}}
 
-## `admonition` shortcode
+## Admonition { #admonition-shortcode }
```

Comment:
Is the heading identifier trying to preserve old bookmarks?

I like the intent but feel like it might be worth it to break it anyway so that we don't have to worry about adding this to every new heading in the future.
I think it should have always been just "Name of shortcode" rather than "`name-of-shortcode` shortcode".

Reply:
Oh, and as you might have found out, `doc-validator` isn't aware of this heading syntax haha.

Reply:
Yes, the heading identifier is basically acting like an alias, but I'm okay with not preserving the old headings.